dbt_semantic_interfaces/protocols/semantic_manifest.py

```diff
@@ -24,5 +25,13 @@ def metrics(self) -> Sequence[Metric]:  # noqa: D
     def project_configuration(self) -> ProjectConfiguration:  # noqa: D
         pass
 
+    @property
+    def saved_queries(self) -> Sequence[SavedQuery]:  # noqa: D
+        """Saved queries defined in the manifest.
+
+        Returns a default value for backward compatibility until the next breaking release.
+        """
+        return ()
```

Comment:
Hmmm 🤔 do we need this return? Nothing should be directly inheriting the protocol, and thus won't inherit this implementation. It's up to the implementation of protocols to define the default behavior to satisfy the typing. We should probably stick to the pattern of just having `pass` in the protocol definitions

Reply:
+1 @QMalcolm  - returning a value from the protocol affords zero protection against breakage, since nothing has to inherit from it, and given that our position is you shouldn't inherit from protocols in the general case I agree we should leave the implementations out of here.

Reply:
Yeah, if nothing is inheriting from it, I'll remove.

Reply:
This still should changed to `pass`

Reply:
Missed earlier, but now updated.